drivers/at86rf2xx/include/at86rf2xx_internal.h

```diff
@@ -111,9 +138,16 @@ void at86rf2xx_sram_read(const at86rf2xx_t *dev, uint8_t offset,
  * @param[in] data      data to copy into SRAM
  * @param[in] len       number of bytes to write to SRAM
  */
+#if defined(MODULE_AT86RFA1) || defined(MODULE_AT86RFR2)
+static inline void at86rf2xx_sram_write(const at86rf2xx_t *dev, uint8_t offset,
+                          const uint8_t *data, size_t len) {
```

Comment:
Please align the parameters of the function (add more spaces before `const uint8_t *data`)